spec/cli_v1_commands.yaml

```diff
@@ -770,6 +788,20 @@ commands:
     usage: supabase functions new <Function name>
     subcommands: []
     flags: []
+  - id: supabase-functions-download
+    title: supabase functions download
+    summary: Download a Function from Supabase
+    description: |
+      Download source code of a Function from the linked Supabase project.
```

Comment:
```suggestion
    summary: Download a function from Supabase
    description: |
      Download the source code for a function from the linked Supabase project.
```

Reply:
Keeping the Function capitalised as it's referring to the name of our product.